.github/CODEOWNERS

```diff
@@ -0,0 +1,79 @@
+/packages/jest-puppeteer-axe                    @WordPress/gutenberg-core @ntwb @nerrad @ajitbohra
+/packages/library-export-default-webpack-plugin @WordPress/gutenberg-core @ntwb @nerrad @ajitbohra
+/packages/npm-package-json-lint-config          @WordPress/gutenberg-core @ntwb @nerrad @ajitbohra
+/packages/postcss-themes                        @WordPress/gutenberg-core @ntwb @nerrad @ajitbohra
+/packages/scripts                               @WordPress/gutenberg-core @ntwb @nerrad @ajitbohra
+
+# UI Components
+/packages/components                            @WordPress/gutenberg-core @chrisvanpatten @ajitbohra @jaymanpandya @nosolosw
+/packages/compose                               @WordPress/gutenberg-core @chrisvanpatten @ajitbohra @jaymanpandya
+/packages/element                               @WordPress/gutenberg-core @chrisvanpatten @ajitbohra @jaymanpandya
+/packages/notices                               @WordPress/gutenberg-core @chrisvanpatten @ajitbohra @jaymanpandya
+/packages/nux                                   @WordPress/gutenberg-core @chrisvanpatten @ajitbohra @jaymanpandya
+/packages/viewport                              @WordPress/gutenberg-core @chrisvanpatten @ajitbohra @jaymanpandya
+
+# Utilities
+/packages/a11y                                  @WordPress/gutenberg-core
+/packages/blob                                  @WordPress/gutenberg-core
+/packages/date                                  @WordPress/gutenberg-core
+/packages/deprecated                            @WordPress/gutenberg-core
+/packages/dom                                   @WordPress/gutenberg-core
+/packages/dom-ready                             @WordPress/gutenberg-core
+/packages/escape-html                           @WordPress/gutenberg-core
+/packages/html-entities                         @WordPress/gutenberg-core
+/packages/i18n                                  @WordPress/gutenberg-core @swissspidy
+/packages/is-shallow-equal                      @WordPress/gutenberg-core
+/packages/keycodes                              @WordPress/gutenberg-core
+/packages/priority-queue                        @WordPress/gutenberg-core
+/packages/token-list                            @WordPress/gutenberg-core
+/packages/url                                   @WordPress/gutenberg-core
+/packages/wordcount                             @WordPress/gutenberg-core
+
+# Extensibility
+/packages/hooks                                 @WordPress/gutenberg-core
+/packages/plugins                               @WordPress/gutenberg-core
+
+# Rich Text
+/packages/format-library                        @WordPress/gutenberg-core
+/packages/rich-text                             @WordPress/gutenberg-core
+
+# PHP
+/lib                                            @WordPress/gutenberg-core
+
+# Documentation
+/docs                                           @WordPress/gutenberg-core @chrisvanpatten @mkaz @ajitbohra @nosolosw
```

Comment:
I would also add `*.md` in the Documentation section.

Reply:
Good point, how about `*/README.md` then?

Reply:
> Good point, how about `*/README.md` then?

I think that could work, sure 👍

Reply:
> > Good point, how about `*/README.md` then?
> 
> I think that could work, sure 👍

On second thought, I'm wondering if it might be better to leave out initially, as I worry it could become quite noisy. Like CHANGELOG, I think it could be that a large number of pull requests would include README changes to document their impact to public-facing APIs, in a way which perhaps loses its effectiveness to notifying the "most correct" parties.

Put another way, we can have much more confidence that a change to a file within `/docs` is part of a pull request oriented toward documentation than one which includes a change to a `README` file.